src/aclk/aclk.c

```diff
@@ -21,6 +21,9 @@
 #include <fcntl.h>
 #endif
 
+#define MQTT_DEFAULT_MAX_BUF_SIZE  (25 * 1024 * 1024)
+#define MQTT_PARENT_MAX_BUF_SIZE  (128 * 1024 * 1024)
```

Comment:
that seems quite a lot at a first sight but true - i do not know about how big parents are you talking about... anyway IMHO knowing the rate at which you can send vs. rate you produce data might be of value?

Reply:
Big ones ~1000 children (if not needed it will not use it anyway) and it will be only for parents